src/windows/com.rs

```diff
@@ -240,14 +240,14 @@ impl SerialPort for COMPort {
     }
 
     fn set_timeout(&mut self, timeout: Duration) -> Result<()> {
-        let milliseconds = timeout.as_secs() * 1000 + timeout.subsec_nanos() as u64 / 1_000_000;
+        let milliseconds = timeout.as_millis();
```

Comment:
`as_millis` is also available. I'm not aware of any breaking difference, let me know if I should revert it back or not